.github/pull_request_template.md

```diff
@@ -0,0 +1,15 @@
+### Description
+[Describe what this change achieves]
+ 
+### Issues Resolved
+[List any issues this PR will resolve]
+ 
+### Check List
+- [ ] New functionality includes testing.
+  - [ ] All tests pass
+- [ ] New functionality has been documented.
+  - [ ] New functionality has javadoc added
+- [ ] Commits are signed per the DCO using --signoff 
```

Comment:
I don't think we should include this line item; its explicitly enforced by the PR DCO check enabled on all opensearch-project repositories and the statement at the bottom